fabric-climbing-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,12 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-climbing-v1",
+  "version": "${version}",
+  "license": "Apache-2.0",
+  "depends": {
+    "fabricloader": ">=0.4.0"
+  },
+  "mixins": [
+    "fabric-climbing-v1.mixins.json"
+  ]
+}
```

Comment:
Please update this module to have the same metadata as other modules. I can see description is missing but maybe others. Author should be FabricMC I believe but refer to other modules. Also make sure to copy the icon.png over